### PR TITLE
dynamic content display in progress

### PR DESCRIPTION
**Pull request by Vanilla-svg**

### CODE REVIEW

**Review by deeppsy — APPROVED** [review]
It looks good. Didn't get a chance to run it yet. But nothing screaming out at me

File: src/pages/Courses.js [hunk 1]
```diff
@@ -19,8 +19,8 @@ function Courses() {
   const courses = [
     {
       id: 1,
-      courseCode: "SSG 551",
-      courseTitle: "Introduction to System Engineering",
+      courseCode: "SSG 549",
+      courseTitle: "Introduction to System Reliability and Maintainability",
```
Comment (deeppsy) [inline src/pages/Courses.js:23]: If you can, can you move this data into a separate file. Most of the other data are located in ./src/data.

After moving, you can can import in this file and continue as normal.

**Review by deeppsy — COMMENTED** [review]
Just this new small url

**Comment by deeppsy** [comment]
There's some conflict that needs fixing

File: src/pages/Courses.js (hunk 1 above)
Comment (Vanilla-svg) [reply src/pages/Courses.js:23]: I was actually thinking of having the courses loaded from the user profile in firebase, I did that for the topics and assessments, I'd just need to set up the collection appropriately. What'd you think?

Comment (deeppsy) [reply src/pages/Courses.js:23]: Ahh, I see. Not bad. That works

**Pull request merged**